plugins/mongoose-audit/package.json

```diff
@@ -21,7 +21,7 @@
     "mongoose": ">=5"
   },
   "devDependencies": {
-    "mongoose": "^5"
+    "mongoose": "^5.13.22"
```

Comment:
Is there a reason for this version bump, right now there's no advantage in it right ? Same with the eslint updates above